api/graphql/reservation_units/reservation_unit_serializers.py

```diff
@@ -1,4 +1,5 @@
 from django.core import validators
+from graphene_django.utils.utils import _camelize_django_str
```

Comment:
Would
```python
from graphene.utils.str_converters import to_camel_case
```
work here? It might be a bit better since it's a public API function.

Reply:
Nice found, quick look and it should be the one to use :+1:

Reply:
changed this :lotus_position_man: